### PR TITLE
test-runner: Add -vcolumns to RDMD

### PR DESCRIPTION
**Pull request by AndrejMitrovic**

### CODE REVIEW

File: tests/runner.d [hunk 1]
```diff
@@ -35,16 +35,17 @@ private int main (string[] args)
...
     foreach (test; dirEntries(UnitPath, SpanMode.shallow))
     {
         writeln("Running test on ", test);
         auto pp = pipeProcess(Args ~ test);
         if (pp.pid.wait() != 0)
         {
-            pp.stdout.byLine.each!(a => writeln("[stdout]\t", a));
-            pp.stderr.byLine.each!(a => writeln("[stderr]\t", a));
+            pp.stdout.byLine.each!(writeln);
+            pp.stderr.byLine.each!(line => stderr.writeln(line));
```
Comment (AndrejMitrovic) [inline tests/runner.d]: I can't seem to do `pp.stderr.byLine.each!(stderr.writeln)` here. I get an unusual error:

```D
/Library/D/dmd/src/phobos/std/algorithm/iteration.d(936,48): Error: need `this` for `writeln` of type `@safe void(char[] _param_0)`
```

**Comment by AndrejMitrovic** [comment]
I mainly did this so I can press F2 and go to the offending line when there's a compilation error.

File: tests/runner.d [hunk 2]
```diff
@@ -35,16 +35,17 @@ private int main (string[] args)
         return 1;
     const Args = ["rdmd", ] ~ args[1 .. $] ~
         importPaths.map!(v => "-I" ~ v).array ~
-        lflags.map!(v => "-L" ~ v).array;
+        lflags.map!(v => "-L" ~ v).array ~
+        "-vcolumns";
```
Comment (Geod24) [inline tests/runner.d]: Move it after `"rdmd"`

File: tests/runner.d (hunk 1 above)
Comment (Geod24) [reply tests/runner.d]: It's not an error message coming from the runner hence it shouldn't go to the runner's `stderr` output.

**Comment by Geod24** [comment]
> I mainly did this so I can press F2 and go to the offending line when there's a compilation error.

Can't you Command+click ?

File: tests/runner.d (hunk 1 above)
Comment (AndrejMitrovic) [reply tests/runner.d]: I mean.. the runner itself should just not error at all.

Comment (Geod24) [reply tests/runner.d]: I guess we could remove it but having the distinction seemed useful originally.

File: tests/runner.d [hunk 3]
```diff
@@ -33,7 +33,7 @@ private int main (string[] args)
     const lflags = getLflags();
     if (!importPaths.length || !lflags.length)
         return 1;
-    const Args = ["rdmd", ] ~ args[1 .. $] ~
+    const Args = ["rdmd", ] ~ "-vcolumns" ~ args[1 .. $] ~
```
Comment (Geod24) [inline tests/runner.d]: I meant:
```suggestion
    const Args = ["rdmd", "-vcolumns", ] ~ args[1 .. $] ~
```

File: tests/runner.d (hunk 1 above)
Comment (Geod24) [reply tests/runner.d]: Okay let's just go with this.

**Comment by AndrejMitrovic** [comment]
Fixed, and removed second commit since I can just `grep` in my build script to strip out the `[stderr]` stuff.

**Review by Geod24 — APPROVED** [review]

**Pull request merged**